spec/meilisearch/index/base_spec.rb

```diff
@@ -107,7 +107,8 @@
     expect(index.options).to eq({
                                   headers: {
                                     'Content-Type' => 'application/json',
-                                    'Authorization' => "Bearer #{MASTER_KEY}"
+                                    'Authorization' => "Bearer #{MASTER_KEY}",
+                                    'User-Agent' => MeiliSearch.qualified_version
```

Comment:
Should be `User-agent: Meilisearch (<version>)`

Reply:
Actually, it should be `User-Agent` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent.

Or your complaint is about the idea to have a method to do that?

Reply:
I'm complaining about the `Meilisearch Ruby (<version>)` where the `Meilisearch Ruby` part is missing, if I'm not wrong